spec/services/processors/blog_posts_full_updater_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 describe Processors::BlogPostsFullUpdater do
-  describe '.call' do
+  describe '#call' do
```

Comment:
why this change? you are calling `Processors::BlogPostsFullUpdater.call` so that's a class method an `.call` is correct